miso-service/src/main/java/uk/ac/bbsrc/tgac/miso/service/impl/DefaultSampleService.java

```diff
@@ -350,6 +357,8 @@ private DetailedSample findOrCreateParent(DetailedSample sample) throws IOExcept
       }
     } else if (isTissueSample(tempParent)) {
       return createParentTissue((SampleTissue) tempParent, sample);
+    } else if (isStockSample(tempParent)) {
```

Comment:
Add test cases for this similar to the existing ones in DefaultSampleServiceTestSuite